src/Pelagos/Bundle/AppBundle/Resources/views/Dataland/index.html.twig

```diff
@@ -42,6 +39,13 @@
 
     {{ add_js(path('fos_js_routing_js', { callback: 'fos.Router.setData' })) }}
 
+    {% if dataset.availabilityStatus == 10 %}
```

Comment:
Rather than using 10, use the contant like: constant('Pelagos\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_PUBLICLY_AVAILABLE')

Reply:
👍